e3/anod/context.py

```diff
@@ -364,6 +364,25 @@ def add_action(data, connect_with=None):
             if connect_with is not None:
                 self.connect(connect_with, data)
 
+        def add_dep(deps, dep, dep_instance):
+            """Add a new dependency in a dependencies dict.
+
+            :param deps: the Anod instance ``deps`` attribute
```

Comment:
FWIW, one thing that's missing in the documentation for me is which dependency's list is being updated.
We could solve that problem very simply by just adding that piece of info in this description. On the other hand, how about passing the spec whose dependencies we want to update instead? For me, I think itll be easier to document, and easier I think to read the code... But if you prefer keeping the code, as is, perhaps change the description to:

    The ``deps`` attribute of *an* Anod instance, to which the new dependency is to be added.